packages/office-addin-project/src/convert.ts

```diff
@@ -15,6 +16,7 @@ export async function convertProject(
   manifestPath: string = "./manifest.xml",
   backupPath: string = "./backup.zip"
 ) {
+  const outputPath: string = "./manifest";
```

Comment:
We should convert to the same location as the xml manifest . . . which may not be "./manifest".  (Note that the only reason the json template had a manifest folder was because of a quirk in the json => xml converter)

Reply:
But what about the fact that the JSON manifest could have multiple files?
Like it can be split up into multiple files (for localization is the easiest example). It does seem nice to have it already in a specific folder where all those files can live.

Reply:
Maybe, but we don't want to enforce a project structure if we can avoid it.  We would also have to change all the references to the manifest location built into the project (like in the package.json) which may not be a big deal, but it still extra work.

It might be nice to put the language specific file in a subdirectory, but that might require updating the manifest references to the lang files and I'm not sure it's required.  I think it's best to leave it to the developer to move things around in the project.  If we receive feedback to the contrary, we can add this later.

Reply:
The outputPath needs to be the location of the input manifest path.